waspc/examples/todoApp/todoApp.wasp

```diff
@@ -29,7 +29,7 @@ app todoApp {
     setupFn: import setup from "@server/serverSetup.js"
   },
   client: {
-    rootComponent: import { App } from "@client/App.jsx",
+    rootComponent: import { App } from "@client/App",
```

Comment:
We wanted to keep the extensions in the `main.wasp` so it's not confusing that we MUST include them in backend but they are OPTIONAL in the client. 🤷‍♂️

Reply:
Changing it back... and the worst thing is: I think it was my suggestion :smile: